pom.xml

```diff
@@ -132,6 +116,11 @@
             <artifactId>workflow-basic-steps</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>pipeline-input-step</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Unused after 10215a67c84fa102ad5d39481a72ab5a06a7d4b7?
```suggestion
```

Reply:
Oops yes forgot to remove it.